app/src/org/commcare/android/view/TabbedDetailView.java

```diff
@@ -100,20 +94,20 @@ public void onPageScrollStateChanged(int arg0) {
         });
     }
 
-    /*
+    /**
      * Populate view with content from given Detail.
      */
     public void setDetail(Detail detail) {
         mMenu.setVisibility(VISIBLE);
     }
 
-    /*
+    /**
      * Get form list from database and insert into view.
      */
-    public void refresh(Detail detail, TreeReference reference, int index, boolean hasDetailCalloutListener) {
-        mEntityDetailPagerAdapter = new EntityDetailPagerAdapter(mContext.getSupportFragmentManager(), detail, index, reference,
-                hasDetailCalloutListener, new ListItemViewStriper(this.mOddColor, this.mEvenColor)
-        );
+    public void refresh(Detail detail, TreeReference reference, int index) {
+        EntityDetailPagerAdapter mEntityDetailPagerAdapter =

```

Comment:
Updated; I was lazy about this because I see this so much in our code that I have started to give up on it. Thanks for pushing on it.